samples/snapshot/pom.xml

```diff
@@ -68,6 +68,16 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud.opentelemetry</groupId>
+      <artifactId>exporter-trace</artifactId>
+      <version>0.31.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.cloud.opentelemetry</groupId>
+      <artifactId>propagators-gcp</artifactId>
+      <version>0.33.0-alpha</version>
+    </dependency>
```

Comment:
I'm not sure we need to declare this, we're not declaring it in ssb, and it doesn't seem to be pulled in transitively.

Reply:
I can take this out but I will have to take out the line where we suggest the XCloudTraceContextPropagator. This isn't required so I am fine with removing it.

Reply:
https://cloud.google.com/trace/docs/setup#force-trace has some detail about `X-Cloud-Trace-Context`. I think it's worth keep in the sample for most customers.

We didn't need it in ssb because ssb is the originator of all traces, and therefore doesn't need to propogate anything.